### ISSUE TITLE
The chip has a function to control LED via SPI, this could be reflected in the library.

### ISSUE THREAD

**Issue by MartinSpengler** [bitcraze/Bitcraze_PMW3901#12]
Hello bitrcaze and thanks a lot for providing this library. As you may know, a lot of people are using this library with several PCB that use the PMW3901 module. Some have LED that are controlled via SPI by the PMW3901. Unfortunately the registers used to control these LED are AFAIK mostly undocumented so that it needs some amount of reverse engineering to use them.

I added a small function to Bitcraze_PMW3901.cpp that controls the LED, if you think it makes sense you could add it to the Arduino lib...

```
void Bitcraze_PMW3901::setLed(bool state)
{
  if(state){
    // trying to switch on the LED
    delay(200);
    registerWrite(0x7f, 0x14);
    registerWrite(0x6f, 0x1c); // DEC 28
    registerWrite(0x7f, 0x00);
  }else{
    // trying to switch off the LED
    delay(200);
    registerWrite(0x7f, 0x14);
    registerWrite(0x6f, 0x00); 
    registerWrite(0x7f, 0x00);
  }
}
```

**Comment by ataffanel** [comment]
Hi, this looks good to me, thanks!

If you want you can do a pull request to this repos and I will merge it and create a new version, otherwise I can add and commit the function myself.

**Comment by dave-hillier** [comment]
I've noticed this hasn't been done so I made a PR. Thanks @MartinSpengler.

**Comment by ataffanel** [comment]
Implemented in #13
